satellite/satellitedb/repairqueue.go

```diff
@@ -32,10 +32,10 @@ func (r *repairQueue) Insert(ctx context.Context, seg *pb.InjuredSegment) (err e
 func (r *repairQueue) Select(ctx context.Context) (seg *pb.InjuredSegment, err error) {
 	defer mon.Task()(&ctx)(&err)
 	err = r.db.QueryRowContext(ctx, `
-		UPDATE injuredsegments SET attempted = timezone('utc', now()) WHERE path = (
```

Comment:
note: postgres now() returns time in the server's local timezone. cockroachdb returns time in utc.